test/AutoRest.Shared.Tests/AutoRest.Shared.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
```

Comment:
lets add net6 as well to match azure-sdk-for-net since there are many shared files between the two repos.

Reply:
Done